devel/ihaskell/Portfile

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           haskell 1.0
+
+github.setup        gibiansky IHaskell 93bfa3a
+git.branch          master
+
+# set the version to be the date of the last commit
+version             2019.08.11
+
+set name_lower      [string tolower ${name}]
+categories          devel haskell
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         A Haskell kernel for IPython.
+long_description    \
+    IHaskell is a kernel for the Jupyter project, which allows you to \
+    use Haskell inside Jupyter frontends (including the console and \
+    notebook). For a tour of some IHaskell features, check out the \
+    demo Notebook at \
+    http://nbviewer.ipython.org/github/gibiansky/IHaskell/blob/master/notebooks/IHaskell.ipynb. More \
+    example notebooks are available on the wiki at \
+    https://github.com/gibiansky/IHaskell/wiki. The wiki also has more \
+    extensive documentation of IHaskell features.
+
+# use these to specify python versions, python3 required 
+set python3_version 3.7
+set python3_version_nickname \
+    [join [lrange [split ${python3_version} .] 0 1] {}]
+
+# See https://github.com/gibiansky/IHaskell#mac,
+# https://github.com/gibiansky/IHaskell/blob/master/requirements.txt
+depends_build-append  \
+                    port:cabal \
+                    port:ghc \
+                    port:pango \
```

Comment:
Can `pango-devel` also work here? If so, use this instead:

`                        path:lib/pkgconfig/pango.pc:pango \
`

Reply:
Done.